.github/workflows/evaluation.yml

```
@@ -44,9 +44,13 @@ jobs:
           lake -R build
 
   extract-goals:
-    name: Check extracted goals
+    name: Extract goals
     runs-on: ubuntu-latest
     needs: eval-build
+    strategy:
+      matrix:
+        extract_offset: [0, 1, 2, 3, 4]
+        extract_stride: [5]
```

```suggestion
        extract_stride: [5]
```

Why is this a list? I am confused. Oh, I seem because it's a matrix? hmm.

---

Indeed. Its a list of values that is executed. We (ab)use this list to document a number, but in the end its still a matrix for which the cross-product of all vectors is run,